src/App.tsx

```diff
@@ -1,14 +1,48 @@
-import React from 'react';
+import React, { useMemo, useState, useCallback } from 'react';
+import classNames from 'classnames';
+import { debounce } from 'lodash';
 import './App.scss';
 import { peopleFromServer } from './data/people';
 
 export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+  const [query, setQuery] = useState('');
+  const [applyQuery, setApplyQuery] = useState('');
+  const [debounceDelay, setDebounceDelay] = useState(1000);
```

Comment:
Move delay to const
![2023-06-30_14h38_24](https://github.com/mate-academy/react_autocomplete/assets/61904995/d41cfa5f-ea47-4779-924a-bd0b8438c63e)